topics/search.md

```diff
@@ -190,4 +220,46 @@ The following list of configurations can be modified at runtime using the `CONFI
    sizes may improve performance by enhancing CPU cache efficiency, but come at the cost of increased memory usage due
    to pre-allocation for potential future growth. (Default: 10K)
 
+2. `search.query-string-bytes`: Controls the maximum length in bytes of the query string for FT.SEARCH commands. 
 
+3. `search.query-string-depth`: Controls the maximum depth of query string parsing for FT.SEARCH commands, preventing 
+   overly complex nested queries. (Default: 1000, Min: 1, Max: UINT_MAX)
+
+4. `search.query-string-terms-count`: Controls the maximum number of terms (nodes in the predicate tree) allowed in a 
+   query string for FT.SEARCH commands. (Default: 16, Min: 1, Max: 32)
+
+5. `search.max-indexes`: Controls the maximum number of search indexes that can be created. 
```

Comment:
Are all of these breaking changes? All of these seem like limits that didn't exist before. Are we releasing these with Valkey 9.0?